tests/zfs-tests/tests/functional/rsend/send_realloc_encrypted_files.ksh

```diff
@@ -78,7 +78,7 @@ for i in {1..$passes}; do
 	# Randomly modify several dataset properties in order to generate
 	# more interesting incremental send streams.
 	rand_set_prop $POOL/fs checksum "off" "fletcher4" "sha256"
-	rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4"
+	rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4" "zstd"
```

Comment:
isn't this a typo?
shouldn't:
`rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4" "zstd"`
be
`rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4" "zstd"`